build-scripts/hack/util.py

```diff
@@ -48,3 +48,9 @@ def read_file(path: Path) -> str:
 
 def read_url(url: str) -> str:
     return urlopen(url).read().decode().strip()
+
+def helm_pull(chart_name: str, repo_url: str, version: str, destination: Path) -> None:
+    parse_output(["helm", "repo", "add", chart_name, repo_url])
+    parse_output(["helm", "pull", f"{repo_url}/{chart_name}", "--version", version, "--destination", destination])
```

Comment:
this was not addressed, use `helm pull $chart --repo $repo`

Reply:
Removed the helm repo add.

Reply:
you also need `--repo $repo_url`, otherwise this will not work